src/promptflow/promptflow/exceptions.py

```diff
@@ -221,6 +223,12 @@ class ValidationException(UserErrorException):
     pass
 
 
+BUILD_IN_USER_EXCEPTIONS = (
+    ImportError, AttributeError, TypeError, ValueError, NotImplementedError, ModuleNotFoundError, IndexError, KeyError,
+    KeyboardInterrupt, SystemError, SystemExit, TypeError, ValueError, FileNotFoundError, PermissionError
+)
```

Comment:
UserError: 
ImportError, KeyboardInterrupt, PermissionError

Reply:
done, in addition, I add "SystemExit".